tests/test_events.py

```
@@ -180,3 +183,77 @@ def test_eventmgr_storage_state(event_io, early_event):
     assert mgr_two.get_chronological_task_events_by_type(early_event.task_name) == {"state": [early_event]}
 
 
+def test_cycle_time_noop():
+    assert data.cycle_time([]) == None
```

Ruff (0.15.6)

[error] Comparison to `None` should be `cond is None`

Replace with `cond is None`

(E711)